yoticlient.go

```diff
@@ -100,22 +95,22 @@ func (client *Client) GetSdkID() string {
 // GetActivityDetails requests information about a Yoti user using the one time use token generated by the Yoti login process.
```

Comment:
It would be nice to mention here that this function may return `yoti.TemporaryErrors`